lib/charms/tls_certificates_interface/v2/tls_certificates.py

```diff
@@ -1073,7 +1077,7 @@ def _on_relation_changed(self, event: RelationChangedEvent) -> None:
         provider_relation_data = _load_relation_data(event.relation.data[self.charm.app])
         if not self._relation_data_is_valid(requirer_relation_data):
             logger.warning(
-                f"Relation data did not pass JSON Schema validation: {requirer_relation_data}"
+                "Relation data did not pass JSON Schema validation: %s", requirer_relation_data
```

Comment:
We probably don't need to specify the content of the relation data in the logs.

Reply:
Agreed, removed.